.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+name: Create a release on pypi.org
+
+on: push
```

```suggestion
on:
  push:
  pull_request:
```
So that PRs changing this workflow are also tested

---

Done.

Super weird that it is
```
on:
  push:
  pull_request:
```
The trailing `:` on the push and pull request line are "unusual", at least I don't see it often :)